### PR TITLE
Update docker run information. Add sell at loss count env variable with default to 1

### PR DESCRIPTION
**Pull request by ChrisCPoirier**

### CODE REVIEW

File: app/mover.js [hunk 1]
```diff
@@ -165,6 +166,11 @@ class Mover {
 
       // If its dropped, then bail
       if (currentReturn < this.config.sell_at_loss_percent) {
+        sell_at_loss_count++
+        if (sell_at_loss_count < this.config.sell_at_loss_count) {
```
Comment (thisiskj) [inline app/mover.js:170]: @ChrisCPoirier Should this be `<=` ?

**Review by thisiskj — DISMISSED** [review]
Take a look at the 1 bug, and if you can add the new config option to the readme

File: app/mover.js (hunk 1 above)
Comment (ChrisCPoirier) [reply app/mover.js:170]: @thisiskj I do not believe so, The intent is that if the sell_at_loss_count is below the config.sell_at_loss_count we `continue` the main loop and recheck prices, else we have matched or exceed our threshold and want to move forward with the sell

**Review by thisiskj — APPROVED** [review]
Great work, again!

**Pull request merged**